tests/console/system_prepare.pm

```diff
@@ -47,6 +48,7 @@ sub run {
         if (is_sle('15+') && check_var('SLE_PRODUCT', 'sles')) {
             add_suseconnect_product(get_addon_fullname('base'), undef, undef, undef, 300, 1);
             add_suseconnect_product(get_addon_fullname('serverapp'), undef, undef, undef, 300, 1);
+            add_suseconnect_product(get_addon_fullname('desktop'), undef, undef, undef, 300, 1) if is_sle('<15', get_var('ORIGIN_SYSTEM_VERSION')) && is_ppc64le;
```

Comment:
a boolean with a string as default value? This module is shared, for us might be inconvenient to test desktop in some cases, but for other squad might be ok, wasn't desktop default in sle 12?

Reply:
For SLE 12, there is no desktop application or sles application. For this part of code in system_prepare.pm, we just try to register modules after media migration(de-registered) to make it work for following regression test. And this change should be seen as a hack for multi-user test on ppc64le. 
I think the current condition is somehow enough, media_migration, sle12 and ppc64le, but if you think not sure whether this will affect other squad, we can define a special condition. WDYT?

Reply:
if `get_var('ORIGIN_SYSTEM_VERS` is undef I guess it would affect other job groups because it would take default value of version inside the function, but others do not use that variable so if you check that that variable is defined then it should be safer, right?

Reply:
Ok, updated to get_required_var, wait VR http://openqa.suse.de/tests/14171938#live

Reply:
that will break the test suite not using that variable, I meant to check for undefined.

Reply:
Updated code and VR.

Reply:
please, read my previous comment.

Reply:
sorry, I think I confused you, it was almost fine in the original version (because the handling of undef is already there), I would narrow to sle 12 sp5 though:
`if is_sle('=12-sp5', get_var('ORIGIN_SYSTEM_VERSION')) && is_ppc64le;`

Reply:
Ok, updated code and VR.